vllm/sequence.py

```diff
@@ -560,16 +563,35 @@ def __eq__(self, other: object) -> bool:
                 and self.prompt_logprobs == other.prompt_logprobs)
 
 
+class EmbeddingSequenceGroupOutput:
```

Comment:
Similar logic here as RequestOutput vs EmbeddingRequestOutput

Reply:
There are discussions in https://github.com/vllm-project/vllm/pull/3187 about changing the RequestOutput to abstract.

Reply:
Done. I added an ABC of RequestOutput